config/routes.rb

```diff
@@ -27,6 +27,7 @@
   namespace :new_admin do
     resources :mentoring, only: :index
     resources :allocation_chart, only: :index
+    resources :allocations, only: %i[new create show], as:"user_allocations"
```

Comment:
you can use symbol  as argument for the `:as` option ` as: :user_allocations` but I don't see reason

Reply:
but if you don't add the add the URL helper name should be new_admin_user_allocations_path right?

Reply:
yeah, with the `as:` it will be like that